cmd/slashing.go

```diff
@@ -0,0 +1,85 @@
+package cmd
+
+import (
+	"strings"
+
+	sdkclient "github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/spf13/cobra"
+)
+
+func slashingParamsCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "params",
+		Short: "Query the current slashing parameters",
+		Args:  cobra.NoArgs,
+		Long: strings.TrimSpace(`Query genesis parameters for the slashing module:
+
+$ <appd> query slashing params
+`),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			client := config.GetDefaultClient()
+			result, err := client.QuerySlashingParams()
+			if err != nil {
+				return err
+			}
+			return client.PrintObject(result)
+		},
+	}
+	return cmd
+}
+
+func slashingSigningInfoCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "signing-info [validator-conspub]",
```

Comment:
Its kinda hard getting the conspub. Is there a way that user can pass a couple of validator identifiers in here (say the validator account, or the pub key in hash format) and we can parse them and return the data?